datahub-web-react/src/app/entity/group/GroupProfile.tsx

```diff
@@ -89,11 +89,20 @@ export default function GroupProfile() {
     // Side bar data
     const sideBarData = {
         photoUrl: undefined,
-        avatarName: data?.corpGroup?.info?.displayName || data?.corpGroup?.name,
-        name: data?.corpGroup?.info?.displayName || data?.corpGroup?.name || undefined,
-        email: data?.corpGroup?.editableProperties?.email || undefined,
+        avatarName:
```

Comment:
actually, here we should use 

entityRegistry.getDisplayName(EntityType.CorpGroup, data?.corpGroup?)

Reply:
We can do this in another PR. for now this lgtm.